index.html

```diff
@@ -3,11 +3,14 @@
     <head>
         <meta charset="utf-8">
         <meta name="viewport" content="width=device-width, initial-scale=1.0, shrink-to-fit=no">
-        <title>CVSS v4.0 calculator – DRAFT</title>
+        <title>CVSS v4.0 calculator</title>
         <script src="https://unpkg.com/vue@3.2.45/dist/vue.global.prod.js"></script>
         <script src="cvss_config.js?v=2"></script>
         <script src="cvss_details.js?v=1"></script>
         <script src="cvss_lookup.js?v=5"></script>
+        <script src="max_composed.js"></script>
+        <script src="max_hamming.js"></script>
+        <link rel="stylesheet" href="styles.css?v=6">
```

Comment:
Line duplicate.

Reply:
I think it would still be a good idea to remove `styles.css` duplicate line.

Reply:
Done